ebb_lint/test/test_flake8.py

```diff
@@ -378,64 +379,67 @@ def f():
 finally:
     do_something_else()
 
-    ''',
+        ''',
 
-    '''
+        '''
 
 class C(object):
     $L212$@staticmethod
     def f():
         pass
 
-    ''',
+        ''',
 
-    '''
+        '''
 
 class C(object):
     @classmethod
     def f(cls):
         pass
 
-    ''',
+        ''',
 
-    '''
+        '''
 
 class C(object):
     f = staticmethod(g)
 
-    ''',
+        ''',
 
-    '''
+        '''
 
 spam  $L303$# noqa
 
-    ''',
+        ''',
 
-    '''
+        '''
 
 spam  # eggs $L303$# noqa
 
-    ''',
+        ''',
 
-    '''
+        '''
 
 $L204$'a' 'b'  $L303$# noqa
 
-    ''',
+        ''',
 
-    '''
+        '''
 repr(
     foo for foo in range(3),
 )
-    ''',
+    ''']
+]
 
-    pre_py35skip('''
+all_sources.extend([
+    pytest.param('''
 def foo(one, two):
     pass
 
 foo(**{'one': 1}, two=2)
-    '''),
+    ''', marks=pre_py35skip)])
```

Comment:
A lot of changes like

    pre_py35skip(s)
to

    pytest.param(s, marks=pre_py35skip)